subsys/net/lib/lwm2m/lwm2m_engine.h

```diff
@@ -144,6 +144,7 @@ const char *lwm2m_engine_get_attr_name(const struct lwm2m_attr *attr);
 int  lwm2m_socket_add(struct lwm2m_ctx *ctx);
 void lwm2m_socket_del(struct lwm2m_ctx *ctx);
 int  lwm2m_socket_start(struct lwm2m_ctx *client_ctx);
-int  lwm2m_parse_peerinfo(char *url, struct sockaddr *addr, bool *use_dtls, bool is_firmware_uri);
+int  lwm2m_parse_peerinfo(char *url, struct sockaddr *addr, bool *use_dtls, char *desthostname,
```

Comment:
I'd rather prefer passing the whole lwm2m_context pointer instead of each field individually.

Reply:
Could be a future improve?

Reply:
It'd be better to make it right all along.

Reply:
I will do it now so.